Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.16-alpine
```

Comment:
You might want to change this, see https://github.com/mrusme/xbscli/blob/9aa886e7b19e9214ba2e9046fee8d363e2689c77/go.mod#L3C1-L3C8

Reply:
updated to 1.19